python/dgl/data/ppi.py

```diff
@@ -136,3 +141,138 @@ def __getitem__(self, item):
             return self.valid_graphs[item], self.features[self.valid_mask_list[item]], self.valid_labels[item]
         if self.mode == 'test':
             return self.test_graphs[item], self.features[self.test_mask_list[item]], self.test_labels[item]
+
+
+class PPIDataset(object):
```

Comment:
How's this different than `LegacyPPIDataset`?

Reply:
Almost same other than the `__getitem__` part. I changed it to inheritance and override `__getitem__`